### PR TITLE
update GetStatVarGroup and GetStatVarGroupNode place -> entity

### PR DESCRIPTION
**Pull request by n-h-diaz**
wanted to double check that it's okay to make this change? 
(I'll update website next to switch places -> entities for this API)

### CODE REVIEW

**Review by shifucun — APPROVED** [review]
Thanks for the update!

File: proto/stat_var.proto [hunk 1]
```diff
@@ -180,16 +181,16 @@ message StatVarGroupNode {
...
   // The stat var or stat var group ID to query for.
   string stat_var_group = 1;
-  // The places that the stat var group is associated with. The response should
-  // only contain the children stat var (group) if it has data for at least one
-  // of the places.
-  repeated string places = 2;
+  // The entities that the stat var group is associated with. The response
+  // should only contain the children stat var (group) if it has data for at
+  // least one of the entities.
+  repeated string entities = 2;
```
Comment (pradh) [inline proto/stat_var.proto:193]: To confirm, stanford doesn't use this API, right?  They wanted to use some SV hierarchy APIs in the past.

Comment (shifucun) [reply proto/stat_var.proto:193]: They don't have public traffic. This is the requests in the past 30 days https://screenshot.googleplex.com/9kihiGV3wFwYmag

Comment (pradh) [reply proto/stat_var.proto:193]: Great, thanks for checking!

Comment (n-h-diaz) [reply proto/stat_var.proto:193]: thank you both :)

**Pull request merged**